<?xml version="1.0" ?>
<sdf version="1.6">
  <world name="air_pressure_sensor">
    <plugin
      filename="libignition-gazebo-air-pressure-system.so"
      name="ignition::gazebo::systems::AirPressure">
    </plugin>

    <model name="air_pressure_model">
      <static>true</static>
      <pose>4 0 3.0 0 0.0 3.14</pose>
      <link name="link">
        <pose>0.05 0.05 0.05 0 0 0</pose>
        <inertial>
          <mass>0.1</mass>
          <inertia>
            <ixx>0.000166667</ixx>
            <iyy>0.000166667</iyy>
            <izz>0.000166667</izz>
          </inertia>
        </inertial>
        <collision name="collision">
          <geometry>
            <box>
              <size>0.1 0.1 0.1</size>
            </box>
          </geometry>
        </collision>
        <visual name="visual">
          <geometry>
            <box>
              <size>0.1 0.1 0.1</size>
            </box>
          </geometry>
        </visual>
        <sensor name="air_pressure_sensor" type="air_pressure">
          <always_on>1</always_on>
          <update_rate>30</update_rate>
          <visualize>true</visualize>
          <air_pressure>
            <reference_altitude>123</reference_altitude>
            <pressure>
              <noise type="gaussian">
                <mean>0</mean>
                <stddev>0</stddev>
              </noise>
            </pressure>
          </air_pressure>
        </sensor>
      </link>
    </model>

  </world>
</sdf>
